lib/components/app/responsive-webapp.js

```diff
@@ -211,6 +212,7 @@ class RouterWrapperWithAuth0 extends Component {
         history={history}>
         <div>
           <Switch>
+            {/* TODO: Review the url schemes for consistency and brevity. */}
```

Comment:
Maybe make an issue or something instead of a comment.

Reply:
I moved the comment to #309.